src/turbomind/models/llama/LlamaV2.cc

```diff
@@ -360,13 +360,8 @@ void LlamaV2<T>::dynamicDecode(int*            token_ids,
         {"local_batch_size", {MEMORY_CPU, TYPE_INT32, {1}, &local_batch_size}},
     };
 
-    const std::vector<std::string> optional_inputs{"stop_words_list",
-                                                   "bad_words_list",
-                                                   "runtime_top_k",
-                                                   "runtime_top_p",
-                                                   "temperature",
-                                                   "repetition_penalty",
-                                                   "random_seed"};
```

Comment:
why remove `random_seed`

Reply:
“random_seed” has no use in dynamic decoder